lib/routes/avatar/upload.js

```diff
@@ -55,6 +59,7 @@ module.exports = {
         return db.addAvatar(id, uid, url, FXA_PROVIDER);
       })
       .done(function uploadDone() {
+        profileUpdatedQueue.send({ uid: uid.toString('hex') });
```

Comment:
It looks like this returns a `Promise`, over in the `sqs.js` file. Should we wait on it before replying, or is ignoring it on purpose? If it is, may be worth a comment acknowledging that so Future Us can remember.